project.clj

```diff
@@ -3,6 +3,16 @@
   :url "https://github.com/weavejester/meta-merge"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]]
-  :aliases {"test-all" ["with-profile" "default:+1.6" "test"]}
-  :profiles {:1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}})
+  :plugins [[lein-cljsbuild "1.0.3"]
+            [lein-doo "0.1.6"]]
+  :aliases {"test-cljs" ["doo" "nashorn" "test" "once"]
+            "test-all" ["do" ["test"] ["test-cljs"]]}
+  :profiles {:provided
+             {:dependencies [[org.clojure/clojure "1.7.0"]
```

Comment:
Whilst we're not directly concerned about uberjars, it is something to consider for others downstream.